ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -1115,6 +1161,8 @@ struct EthSwapDetails: Equatable {
 
   /// Token being swapped to
   let toToken: BraveWallet.BlockchainToken?
+  /// To token network (not yet being used. preparation for bridging)
+  let toNetwork: BraveWallet.NetworkInfo
```

Comment:
nit: unneeded comment
```suggestion
  /// To token network (not yet being used. preparation for bridging)
  let toNetwork: BraveWallet.NetworkInfo
```